assets/index.js

```diff
@@ -41,7 +47,53 @@ function updateTimer() {
 		}
 		return response.json()
 	}).then(capturing => {
-		console.debug('capturing', capturing)
+		console.log('capturing', capturing);
+		if(is_active != capturing && !capturing){
+			updateCalendar()
+		}
+		is_active = capturing;
 		updateView(capturing ? config.capturing : config.idle);
 	})
 }
+
+function parseCalendar(active){
+	// Do we want 'Startet/Endet in' or 'Startet/Endet um'?
+	let diff = 0;
+	let t = 0;
+
+	now = Date.now();
+	if (calendar.length > 0){
+		t = is_active ? calendar[0].End : calendar[0].Start;
+		diff = t - now;
+		console.debug('Diff ', diff, t, now);
+	} else {
+		console.debug('Calendar is empty');
+		
+	}
+
+	hours = (diff > 0) ? Math.floor(diff / (1000 * 60 * 60)) : 0;
+	minutes = (diff > 0) ? Math.floor((diff / (1000 * 60)) % 60) : 0;
+	seconds = (diff > 0) ? Math.floor((diff / 1000) % 60) : 0;
+
+	hours = (hours < 10) ? '0' + hours : hours;
+	minutes = (minutes < 10) ? '0' + minutes : minutes;
+	seconds = (seconds < 10) ? '0' + seconds : seconds;
+
```

Comment:
Maybe do something like this which you could then use in logging and the display:

```suggestion
const time_remaining = `${hours}:${minutes}:${seconds}`;
```

Reply:
Sounds like a good idea!